### ISSUE TITLE
Panic when a list page's route is empty

### ISSUE THREAD

**Issue by dominikbraun** [verless/verless#192]
For each directory inside the `content` directory, verless automatically creates an overview page containing all pages inside that directory. This overview page is called list page, represented by the type `model.ListPage`.

Each page - and a list page is no exception - needs to have a valid `Route` field: `/` for the top-level list page, `/blog` for the list page for a `blog` directory and so on.

If this `Route` field is _not_ set, this is a programming error and verless shall panic:

https://github.com/verless/verless/blob/7fc7f50defdd734b17699e649b723fff3d23b8e3/writer/writer.go#L66

After accessing the list page, we should check if `lp.Route == ""` and panic if it is.

**Why?** Because `ListPage.Route` represents the actual path where the list page will be rendered in the filesystem. If that route is empty, the filesystem path is `""` and it will be rendered as top-level list page, which is incorrect.

**Comment by hmble** [comment]
What should be the panic message ?
```go
if lp.Route == "" {
   panic("route should not be empty")
}
```

is this okay ?

**Comment by dominikbraun** [comment]
Yes, or you even may write "route must not be empty" as this really is a strict requirement.

**Comment by dominikbraun** [comment]
Fixed with #195.
